src/en/charms-storage.md

```diff
@@ -18,49 +23,50 @@ management abilities. Related commands are listed below, along with a brief
 description of each.
 
 [`add-storage`][commands-add-storage]
-: Adds unit storage dynamically.
+: Creates and attaches a storage instance to a unit.
 
 [`attach-storage`][commands-attach-storage]
-: Attaches existing storage to a unit.
+: Attaches an existing storage instance to a unit.
 
 [`create-storage-pool`][commands-create-storage-pool]
 : Creates or defines a storage pool.
 
 [`detach-storage`][commands-detach-storage]
-: Detaches storage from a unit.
+: Detaches a storage instance from a unit.
+
+[`remove-storage`][commands-remove-storage]
+: Removes a storage instance from a model.
 
 [`show-storage`][commands-show-storage]
-: Shows the details of a specific storage instance.
+: Shows details of a storage instance.
 
 [`storage`][commands-storage]
-: Lists details of all storage instances in the model.
+: Lists all storage instances in a model.
 
 [`storage-pools`][commands-storage-pools]
-: Lists storage pools.
+: Lists all storage pools in a model.
 
-[`remove-storage`][commands-remove-storage]
-: Removes storage from the model.
+## Storage constraints
 
-## Deploying a charm with storage options
+Several properties are used to dictate how storage is allocated:
 
-The advanced storage features of Juju naturally depend upon charms that have
-storage options. This document will focus on the
-[PostgreSQL charm][postgresql-charm] which uses these features to store the
-database contents separately from the root filesystem.
+- 'pool': class of storage (e.g. magnetic, SSD)
+- 'size': size of each volume
+- 'count': number of volumes
 
-### Storage constraints
+The default pool (e.g. 'ebs' for AWS, 'cinder' for OpenStack) is given by:
```

Comment:
So above "pool" is described as "magnetic/ssd" but here it's "ebs" which can be both magnetic and ssd. This seems a bit confusing.